ipywidgets/widgets/widget.py

```diff
@@ -388,7 +386,7 @@ def get_view_spec(self):
 
     _view_count = Int(None, allow_none=True,
         help="EXPERIMENTAL: The number of views of the model displayed in the frontend. This attribute is experimental and may change or be removed in the future. None signifies that views will not be tracked. Set this to 0 to start tracking view creation/deletion.").tag(sync=True)
-    comm = Instance('ipykernel.comm.Comm', allow_none=True)
+    comm = Instance(Comm, allow_none=True)
```

Comment:
We should check that there was not a reason for using the string version.

In that case, it would be `ipywidgets.kernel.Comm`

Reply:
Could you explain why ipywidgets.kernel.Comm?

Reply:
Because I moved the import of `Comm` in `ipywidgets.kernel`. I can change it to `'ipywidgets.kernel.Comm'` like Sylvain suggested, but `Comm` is imported in this file so it doesn't make a big difference.